{
  "results": {
    "tool": {
      "name": "cypress"
    },
    "summary": {
      "tests": 42,
      "failed": 42,
      "passed": 0,
      "skipped": 0,
      "pending": 0,
      "other": 0,
      "start": 1723561090316,
      "stop": 1723561134839
    },
    "tests": [
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 1085,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/16/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 758,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/16/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 735,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/16/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 779,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/16/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 764,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/16/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 830,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/16/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/16/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/16/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 781,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/17/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 765,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/17/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 758,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/17/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 737,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/17/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 755,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/17/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 751,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/17/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/17/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/17/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 773,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/18/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 746,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/18/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 749,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/18/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 815,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/18/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 750,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/18/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 751,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/18/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/18/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/18/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 791,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/19/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 768,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/19/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 767,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/19/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 794,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/19/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 761,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/19/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 742,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/19/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/19/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/19/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 810,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/2/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 756,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/2/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 749,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/2/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 791,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/2/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 745,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/2/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 761,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/2/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/2/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/2/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 817,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/20/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 750,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/20/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 757,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/20/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 807,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/20/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 733,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/20/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 714,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/20/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/20/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/20/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page displays counter and two buttons",
        "status": "failed",
        "duration": 786,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/conter.cy.ts:9:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/3/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up",
        "status": "failed",
        "duration": 759,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/conter.cy.ts:16:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/3/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Counter Page count up and down",
        "status": "failed",
        "duration": 747,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/conter.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/conter.cy.ts:25:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/3/conter.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page displays list with enabled add button",
        "status": "failed",
        "duration": 767,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/itemslist.cy.ts:12:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/3/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page add button adds an item",
        "status": "failed",
        "duration": 733,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/itemslist.cy.ts:19:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/3/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      },
      {
        "name": "Items List Page remove button removes an item",
        "status": "failed",
        "duration": 744,
        "message": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "trace": "CypressError: `cy.visit()` failed trying to load:\n\nhttp://next-app:3000/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND next-app\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131916:74)\n    at visitFailedByErr (http://localhost:64626/__cypress/runner/cypress_runner.js:131315:12)\n    at <unknown> (http://localhost:64626/__cypress/runner/cypress_runner.js:131915:11)\n    at tryCatcher (http://localhost:64626/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:64626/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:64626/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:64626/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:64626/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:64626/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:64626/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:64626/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at openPage (webpack:///./cypress/e2e/3/itemslist.cy.ts:4:0)\n    at Context.eval (webpack:///./cypress/e2e/3/itemslist.cy.ts:29:0)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND next-app\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)",
        "rawStatus": "failed",
        "type": "e2e",
        "filePath": "cypress/e2e/3/itemslist.cy.ts",
        "retries": 0,
        "flaky": false,
        "browser": "electron 118.0.5993.159"
      }
    ]
  }
}
